src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media/Given_BasicAutomatedTransformation.cs

```diff
@@ -0,0 +1,236 @@
+			await Arrange(SUT);
+
+			// Top left
+			Assert(0, 0, colored);
+
+			// Bottom left
+			Assert(28 - i, 72 + i, NoColor);
+			Assert(30 + i, 70 - i, colored);
+
+			// Top right
+			Assert(72 + i, 28 - i, NoColor);
+			Assert(70 - i, 30 + i, colored);
+
+			// Bottom right
+			Assert(99, 99, colored);
+		}
+
+		[TestMethod]
+		public async Task When_Scale()
+		{
+			const string colored = "#008000";
+
+			var grid = new Basics_Automated_Transformation();
+			var SUT = grid.Get_ScaleHost();
+
+			await Arrange(SUT);
+
+			// Top left
+			Assert(9 - i, 9 - i, NoColor);
+			Assert(10 + i, 10 + i, colored);
+
+			// Bottom left
+			Assert(9 - i, 90 + i, NoColor);
+			Assert(10 + i, 89 - i, colored);
+
+			// Top right
+			Assert(89 - i, 10 + i, colored);
+			Assert(90 + i, 9 - i, NoColor);
+
+			// Bottom right
+			Assert(89 - i, 89 - i, colored);
+			Assert(90 + i, 90 + i, NoColor);
+		}
+
+		[TestMethod]
+		public async Task When_Composite()
+		{
+			const string colored = "#0000FF";
+			
+			var grid = new Basics_Automated_Transformation();
+			var SUT = grid.Get_CompositeHost();
+			await Arrange(SUT);
+
+			// Center
+			Assert(50, 50, colored);
+
+			// Top left
+			Assert(41 - i, 0, NoColor);
+			Assert(42 + i, 0, colored);
+
+			// Bottom left
+			Assert(41 - i, 70 - i, NoColor);
+			Assert(42 + i, 70 - i, colored);
+			Assert(42, 73 + i, NoColor);
+
+			// Top right
+			Assert(87, 56 - i, NoColor);
+			Assert(87 - i, 59 + i, colored);
+			Assert(88 + i, 59 + i, NoColor);
+
+			// Bottom right
+			Assert(87 - i, 99, colored);
+			Assert(88 + i, 99, NoColor);
+		}
+
+		[Test]
+		public async Task When_Group()
+		{
+			const string colored = "#A000C0";
+			
+			var grid = new Basics_Automated_Transformation();
+			FrameworkElement SUT = grid.Get_GroupHost();
+			await Arrange(SUT);
+
+			// Center
+			Assert(50, 50, colored);
+
+			// Top left
+			Assert(41 - i, 0, NoColor);
+			Assert(42 + i, 0, colored);
+
+			// Bottom left
+			Assert(41 - i, 70 - i, NoColor);
+			Assert(42 + i, 70 - i, colored);
+			Assert(42, 73 + i, NoColor);
+
+			//Topp right
+			Assert(87, 56 - i, NoColor);
+			Assert(87 - i, 59 + i, colored);
+			Assert(88 + i, 59 + i, NoColor);
+
+			// Bottom right
+			Assert(87 - i, 99, colored);
+			Assert(88 + i, 99, NoColor);
+		}
+
+		private async Task Arrange(FrameworkElement SUT)
+		{
+			_sut = SUT;
+			WindowHelper.WindowContent = _sut;
+			WindowHelper.WaitForLoaded(_sut);
+			var renderer = new RenderTargetBitmap();
+			await WindowHelper.WaitForIdle();
+			await renderer.RenderAsync(_sut);
+			_result = await RawBitmap.From(renderer, _sut);
+			WindowHelper.WaitForIdle();
+		}
+
+		private void Assert(float x, float y, string color)
+		{
+			float b = 3; // border
+			float dpiScale = (float)_sut.ActualWidth / (b + 100 + b);
+
+			x *= dpiScale;
+			y *= dpiScale;
+			b *= dpiScale;
+
+			x = x + b;
+			y = y + b;
+
+			ImageAssert.HasColorAt(_result, x, y, color, tolerance: 25);
+		}
+
+	}
+#endif
```

Comment:
@jeromelaban I kept the original test structure that has a custom Assert(float, float , string) method. Because of that, I am having difficulties using `Assert.Inconclusive()`. Do I a) change the name of the current Assert(float, float , string) method for something like Test(float, float, string), b) use #if !__WASM__ c) you know how to make both Asserts work.